### PR TITLE
Editing shipping address / shipping method doesn't update cart

### PR DESCRIPTION
**Pull request by dabe5651**
## Description
The cart isn't updating when you enter shipping address or shipping method. Changed both forms handle submit methods to use getCartDetails action.

## Related Issue
https://github.com/adobe/aem-core-cif-components/issues/283

## How Has This Been Tested?
Tested locally, passed ci

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [x] All unit tests pass on CircleCi.
- [x] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by dabe5651** [comment]
looking over it, I removed a error catch on setShippingAddressesOnCart. Does this need to be added back?

File: react-components/src/components/Checkout/editableForm.js [hunk 1]
```diff
@@ -66,23 +65,16 @@ const EditableForm = props => {
...
-            cartDispatch({ type: 'beginLoading' });
-            cartDispatch({ type: 'endEditing' });
-            if (!isSignedIn) {
-                setGuestEmailOnCart({ variables: { cartId: cartId, email: formValues.email } });
-            }
-        },
-        [dispatch, setShippingAddressesOnCart]
-    );
+    const handleSubmitAddressForm = async formValues => {
+        cartDispatch({ type: 'beginLoading' });
```
Comment (herzog31) [inline react-components/src/components/Checkout/editableForm.js:69]: Regarding the error handling. Yes there should be proper error handling. You could do something like this:

```js
cartDispatch({ type: 'beginLoading' });
try {
    await (setShippingAddressesOnCart({ variables: { cartId: cartId, countryCode: 'US', ...formValues } }));
    await (getCartDetails({ cartDetailsQuery, dispatch: cartDispatch, cartId }));
} catch(err) {
    cartDispatch({ type: 'error', error: err.toString() });
} finally {
    cartDispatch({ type: 'endLoading' });
}
```

**Comment by dabe5651** [comment]
Added error handling but I'm running into issues with these tests that run lint or prettier. I'm having this issue with tests in react-components and ui.apps projects. Everything should be formatted correctly and the tests pass if I remove lint and prettier form the scripts in package.json. 

`PS C:\Repos\ADOBE\aem-core-cif-components\react-components> npm run test

> @adobe/aem-core-cif-react-components@1.0.0 test C:\Repos\ADOBE\aem-core-cif-components\react-components
> npm run lint && npm run prettier:check && jest --coverage


> @adobe/aem-core-cif-react-components@1.0.0 lint C:\Repos\ADOBE\aem-core-cif-components\react-components
> eslint 'src/**/*.js'


Oops! Something went wrong! :(

ESLint: 6.8.0.

No files matching the pattern "'src/**/*.js'" were found.
Please check for typing mistakes in the pattern.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @adobe/aem-core-cif-react-components@1.0.0 lint: `eslint 'src/**/*.js'`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the @adobe/aem-core-cif-react-components@1.0.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.`

**Comment by herzog31** [comment]
@dabe5651 Since you're running on Windows, can you try to use backslashes in the paths for the linting and formatting commands?

**Comment by dabe5651** [comment]
I tried that @herzog31. If this is likely an issue with pc vs mac I can use a mac for committing in the future.

**Comment by herzog31** [comment]
@dabe5651 Alright. The changes look good to me. If you could make your branch write-able to reviewers, I can apply the formatting. Or I could also copy your branch to one in this repo and do the same. And then we can merge your contribution :)

**Comment by dabe5651** [comment]
@herzog31 I invited you and @dplaton to be collaborators. Is this what you meant, or is there a specific setting to allow any reviewers of PRs write access?

**Comment by dplaton** [comment]
@dabe5651 I pulled the branch locally, fixed the linting errors and run the tests. Everything should be fine now.

**Comment by dabe5651** [comment]
thanks, I'll remove that empty ci config I added to my repo most likely causing these issues.

**Pull request merged**